src/style.css

```diff
@@ -0,0 +1,120 @@
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+* {
+  font-family: Avenir, "Avenir-Heavy", sans-serif;
+  box-sizing: border-box;
+  font-weight: 300;
+}
```

Comment:
- always set font styles to the `html` tag and inherit where needed
- in `*` tag only `box-sizing` can be used, anything else is forbidden